pkg/kubectl/kubectl.go

```diff
@@ -108,14 +108,14 @@ func expandResourceShortcut(resource string) string {
 		"ep":     "endpoints",
 		"limits": "limitRanges",
```

Comment:
missed that, done.

On Wed, Jul 15, 2015 at 10:59 AM, Nikhil Jindal <notifications@github.com>
wrote:

> In pkg/kubectl/kubectl.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11313#discussion_r34708635>
> :
>
> > @@ -108,14 +108,14 @@ func expandResourceShortcut(resource string) string {
> >  		"ep":     "endpoints",
> >  		"limits": "limitRanges",
>
> While you are at it, tidy this one as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11313/files#r34708635>
> .
>